src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+function showNotification(message, cls) {
+  const div = document.createElement('div');
+  const body = document.querySelector('body');
+
+  div.classList.add(cls);
+  div.textContent = message;
+  body.append(div);
+};
+
+const promise1 = Promise.resolve(() => {});
```

Comment:
you resolve your promise right away, while it should be resolved on click on logo

Reply:
@qoa11a done